lib/src/telegram/data/main_button_params.dart

```diff
@@ -3,11 +3,25 @@ import 'package:js/js.dart';
 @JS()
 @anonymous
 class MainButtonParams {
```

Comment:
@OlegNovosad these variables are in camel case in js: https://core.telegram.org/bots/webapps#mainbutton

Reply:
Fixed. Then it was not working before as well. Thanks for pointing out!